### PR TITLE
homebank 5.1.5

### PR DESCRIPTION
**Pull request by AAS**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/homebank.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Homebank < Formula
   desc "Manage your personal accounts at home"
   homepage "http://homebank.free.fr"
-  url "http://homebank.free.fr/public/homebank-5.1.4.tar.gz"
-  sha256 "0af35f848c9be864db838868c14977e3f873b54b20be24b6b30189766c5afd1f"
+  url "http://homebank.free.fr/public/homebank-5.1.5.tar.gz"
+  version "5.1.5"
```
Comment (fxcoudert) [inline Formula/homebank.rb]: Please remove the `version` line, which is unnecessary.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

**Comment by fxcoudert** [comment]
FYI: I've reported the fact that `install-sh` is not executable as a bug to upstream https://bugs.launchpad.net/homebank/+bug/1691506

**Comment by AAS** [comment]
I've removed version line as requested.

**Review by fxcoudert — APPROVED** [review]

**Comment by fxcoudert** [comment]
Thank you @AAS for the pull request!

**Pull request closed without merging**